examples/README.md

```diff
@@ -173,7 +172,46 @@ wandb.login()
 
 Whenever you use `Trainer` or `TFTrainer` classes, your losses, evaluation metrics, model topology and gradients (for `Trainer` only) will automatically be logged.
 
-When using 🤗 Transformers with PyTorch Lightning, runs can be tracked through `WandbLogger`. Refer to related [documentation & examples](https://docs.wandb.com/library/integrations/lightning).
+Advanced configuration is possible by setting environment variables:
+
+<table>
+  <thead>
+    <tr>
+      <th style="text-align:left">Environment Variables</th>
+      <th style="text-align:left">Options</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="text-align:left">WANDB_LOG_MODEL</td>
+      <td style="text-align:left">Log the model as artifact at the end of training (<b>false</b> by default)</td>
+    </tr>
+    <tr>
+      <td style="text-align:left">WANDB_WATCH</td>
+      <td style="text-align:left">
+        <ul>
+          <li><b>gradients</b> (default): Log histograms of the gradients</li>
+          <li><b>all</b>: Log histograms of gradients and parameters</li>
+          <li><b>false</b>: No gradient or parameter logging</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left">WANDB_PROJECT</td>
+      <td style="text-align:left">Organize runs by project</td>
+    </tr>
+    <tr>
+      <td style="text-align:left">WANDB_DISABLED</td>
+      <td style="text-align:left">Set to <b>true</b> to disable logging entirely</td>
```

Comment:
I would avoid documenting this env variable as it will be deprecated pretty soon (it's been on my TODO for a little bit): there is now a `report_to` argument in the `TrainingArguments` where you can select the integrations you want to use.

For now the default is all of them but in v5, we will switch to none, so it's a good habit to start using this flag :-)

Reply:
Great to know!

Reply:
Sure, if you want to!

Reply:
Ok I added `report_to`. As for `WANDB_DISABLED` it seems to be currently the only way to disable wandb when it's installed and I know some people sometimes use it for tests. Maybe we should remove it when you set `report_to` to `None`.

Reply:
Yeah I deprecated it in #10100, will remove it in v5 once the switch of default is done.

Reply:
So if you just remove those two lines here, we'll be ready to merge!